### PR TITLE
Remove duplicated dependencies

### PR DESCRIPTION
**Pull request by nauaros**
## Summary
1. Remove duplicated dependencies due to `Adyen3DS2` and `AdyenActions` frameworks.
2. Discard dependency with `AdyenWeChatPay`.
3. Update Bundle IDs in order to submit build to App Store Connect.

### CODE REVIEW

File: Adyen.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5965,9 +5954,9 @@
 				);
 				MARKETING_VERSION = 4.6.1;
 				OTHER_SWIFT_FLAGS = "-Xfrontend -warn-long-expression-type-checking=100";
-				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.AdyenUIHost;
+				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.CheckoutDemoUIKit;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "HostUI Testing with ApplePay";
+				PROVISIONING_PROFILE_SPECIFIER = "Checkout Demo UIKit Distribution";
```
Comment (descorp) [inline Adyen.xcodeproj/project.pbxproj:5959]: Don't commit this 🤔

File: Adyen.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5965,9 +5954,9 @@
 				);
 				MARKETING_VERSION = 4.6.1;
 				OTHER_SWIFT_FLAGS = "-Xfrontend -warn-long-expression-type-checking=100";
-				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.AdyenUIHost;
+				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.CheckoutDemoUIKit;
```
Comment (descorp) [inline Adyen.xcodeproj/project.pbxproj:5957]: Don't commit this

**Review by descorp — CHANGES_REQUESTED** [review]

File: Adyen.xcodeproj/project.pbxproj (hunk 2 above)
Comment (descorp) [reply Adyen.xcodeproj/project.pbxproj:5957]: well, we will have to switch it every time locally to debug, right 🤔

Comment (nauaros) [reply Adyen.xcodeproj/project.pbxproj:5957]: It shouldn't any effect at all. How does it alter debugging?

Comment (descorp) [reply Adyen.xcodeproj/project.pbxproj:5957]: ok, I guess you right.
Distribution certificate doesn't mean that we can't build it in DEBUG mode..

**Review by descorp — APPROVED** [review]

File: Adyen.xcodeproj/project.pbxproj (hunk 1 above)
Comment (descorp) [inline Adyen.xcodeproj/project.pbxproj:5959]: Screenshot 2022-01-13 at 13 03 26

Comment (descorp) [reply Adyen.xcodeproj/project.pbxproj:5959]: Maybe we should use another / refreshed certificate ?

Comment (nauaros) [reply Adyen.xcodeproj/project.pbxproj:5959]: Resolved. Talked offline

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**